valheim-backup

```diff
@@ -50,14 +50,27 @@ backup() {
         debug "No Valheim worlds to backup"
         return
     fi
-    backup_file="$BACKUPS_DIRECTORY/worlds-$(date +%Y%m%d-%H%M%S).zip"
-    pre_backup_hook "$backup_file"
-    info "Backing up Valheim server worlds to $backup_file"
+
     mkdir -p "$BACKUPS_DIRECTORY"
     chmod "$BACKUPS_DIRECTORY_PERMISSIONS" "$BACKUPS_DIRECTORY"
-    zip -r "$backup_file" "worlds/"
-    chmod "$BACKUPS_FILE_PERMISSIONS" "$backup_file"
-    post_backup_hook "$backup_file"
+
+    # Wenn BACKUPS_ZIP true, dann zip
+    if [ ${BACKUPS_ZIP} = true ]; then
+        backup_file="$BACKUPS_DIRECTORY/worlds-$(date +%Y%m%d-%H%M%S).zip"
+        pre_backup_hook "$backup_file"
+        info "Backing up Valheim server worlds to $backup_file"
+        zip -r "$backup_file" "worlds/"
+        chmod "$BACKUPS_FILE_PERMISSIONS" "$backup_file"
+        post_backup_hook "$backup_file"
+    else
+        find worlds/ -name "${WORLD_NAME}.db" -type d | fatal "Could not find world file for world ${WORLD_NAME}"
+        backup_file="$BACKUPS_DIRECTORY/${WORLD_NAME}-$(date +%Y%m%d-%H%M%S).db"
+        pre_backup_hook "$backup_file"
+        info "Backing up Valheim server worlds to $backup_file"
+        cp -v worlds/${WORLD_NAME}.db ${backup_file}
```

Comment:
It would make sense to also copy the backup db files as those are in a consistent state. This db file might be in the process of being written to when you copy it hence resulting in a broken backup file.

Reply:
Should we then just copy the .db.old file instead of the .db file, or both?

Reply:
I mean both likely makes sense. The .db file is newer so that's likely the one you'd want to restore from. However there's a slight chance it's corrupted if the backup runs mid-world-save, whereas the .old should always be in a consistent state.